irohad/ametsuchi/impl/rocksdb_indexer.cpp

```diff
@@ -59,13 +57,15 @@ void RocksDBIndexer::txPositions(
     TxPosition const &position) {
   RocksDbCommon common(db_context_);
 
-  std::string h_hex;
-  h_hex.reserve(hash.hex().size());
+  if (auto res =
+          forTransactionByPosition<kDbOperation::kCheck, kDbEntry::kMustExist>(
+              common, account, ts, position.height, position.index);
+      expected::hasValue(res))
+    return;
 
-  for (auto const c : hash.hex()) h_hex += std::tolower(c);
-
-  common.valueBuffer().assign(
-      fmt::format("{}#{}", asset_id ? *asset_id : "", h_hex));
+  std::string h_hex;
```

Comment:
I am not sure if there any reason of this line. Just return new string from toLower(). [minor]